packages/nifti-volume-loader/src/helpers/scaleNiftiArray.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Given a pixel array, rescale the pixel values using the rescale slope and
+ * intercept
+ *
+ * Todo: add the scaling of PT and SUV
+ * @param array - The array to be scaled.
+ * @param scalingParameters - The scaling parameters
+ * @returns The array being scaled
+ */
+export default function scaleNiftiArray(
+  array: Float32Array | Int16Array | Uint8Array,
+  niftiHeader
+): void {
+  const arrayLength = array.length;
+  const { scl_slope, scl_inter } = niftiHeader;
+
+  if (scl_slope === 0 || Number.isNaN(scl_slope)) {
```

Comment:
Can you check !scl_slope || ...  so that if it ends up null/endefined, you still skip out here?